auslib/blobs/apprelease.py

```diff
@@ -908,6 +916,61 @@ def getReferencedReleases(self):
         return referencedReleases
 
 
+class AdditionalPatchAttributesXMLMixin(object):
```

Comment:
Although the method is called _getAdditionalPatchAttributes, this specific class is for proofs, so ProofXMLMixin is still the right name for it. Please let me know if that doesn't make sense to you, I can see how it's more than a little confusing.